docling_eval/prediction_providers/docling_provider.py

```diff
@@ -108,12 +108,21 @@ def predict(self, record: DatasetRecord) -> DatasetRecordWithPrediction:
             record, res.document, None, res.timings
         )
         pred_record.predicted_segmented_pages = {
-            p.page_no: p.parsed_page for p in res.pages if p.parsed_page is not None
+            p.page_no: self.set_word_cells(p) for p in res.pages if p.parsed_page
         }
         pred_record.status = res.status
 
         return pred_record
 
+    def set_word_cells(self, page: Page):
+        if page.parsed_page is None:
+            _log.warning(
+                f"Page {page.page_no} has no parsed_page, cannot set word_cells."
+            )
+            return page.parsed_page
+        page.parsed_page.word_cells = page.cells
```

Comment:
This is going to inject _fake_ cells into the word_cells of a parsed_page. The only case in which that is currently a valid case is when the document was fully OCRed, hence this is a very partial workaround. 

Also please consider that...
- The standard OCR options in Docling are generating line-level cells (TextCellUnit.LINE), not word-level cells. So this would not compare.
- The `parsed_page` will be populated from Docling, this code would simply overwrite the word_cells without checking if some are present (they will be present whenever the source PDF hat programmatic cells)